python/whylogs/core/constraints/factories/frequent_items.py

```diff
@@ -1,8 +1,30 @@
 from whylogs.core.configs import SummaryConfig
+from whylogs.core.relations import Require
 
 from ..metric_constraints import MetricConstraint, MetricsSelector
 
 
+def is_unique(column_name: str) -> MetricConstraint:
+    """Checks that there are no duplicate values in a column.
+
+    Parameters
+    ----------
+    column_name : str
+        Column the constraint is applied to
+    """
+
+    def unique(metric) -> bool:
+        frequent_strings = metric.to_summary_dict(SummaryConfig())["frequent_strings"]
+        return frequent_strings[0].est == 1
```

Comment:
I think this breaks on large inputs, you probably need to use something like asserting the lower bound estimate is 1.